Palach/Views/PlaygroundView.swift

```diff
@@ -72,4 +63,16 @@ struct PlaygroundView: View {
     func stop() {
         terminalState.terminateProcess()
     }
+
+    func updateLanguagueState() {
+        Task {
+            languageStatus = try! await selectedLanguage.type().detect()
+        }
+    }
+
+    func onSelectedLanguageChange(_ newValue: LanguageOption) {
+        terminalState.reset()
```

Comment:
There was a implicit reset of the terminalState before, because previously the view got entirely destroyed and that is now not happening anymore, but to ensure the same behaviour as before I added the explicit reset here.

If it's not needed I can remove it.